phat/contracts/action_offchain_rollup/lib.rs

```diff
@@ -189,19 +191,19 @@ mod action_offchain_rollup {
         #[ink(message)]
         pub fn set_brick_profile_address(&mut self, brick_profile: AccountId) -> Result<()> {
             self.ensure_owner()?;
-            self.brick_profile = brick_profile;
+            self.brick_profile = BrickProfileRef::from_account_id(brick_profile);
             Ok(())
         }
 
-        /// Get client contract address and RPC endpoint (only owner).
+        /// Get client contract address (only owner).
```

Comment:
How about making it public? Much easier to debug

Reply:
Done